### PR TITLE
Form function

### PR DESCRIPTION
**Pull request by leonardoeng13**
Added the contact form function for this template #5.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -337,13 +338,14 @@ <h1>Get Started</h1>
                     <div class="email">
                         <label for="email">E-mail</label>
                         <br>               
-                        <input type="text" id="email" name="email" placeholder="Your e-mail..">
+                        <input type="text" id="email" name="_replyto" placeholder="Your e-mail..">
```
Comment (hemantso) [inline index.html]: - [x] The `email` input should have an `email` type attribute and `required` attribute to activate the validation on the form submit. Kindly fix it.

File: index.html [hunk 2]
```diff
@@ -327,7 +328,7 @@ <h1>Get Started</h1>
         </div>
 
         <div id="get-started" class="form">
-            <form action=".">
+            <form action="https://formspree.io/f/myyldona" method="POST">
                 <div class="formcontainer">
                     <div class="fistname">
                         <label for="fname">First Name</label>
```
Comment (hemantso) [inline index.html:334]: - [x] The `name` input should have a `max length` attribute and `required` attribute to activate the validation on the form submit. Kindly fix it.

File: index.html [hunk 3]
```diff
@@ -337,13 +338,14 @@ <h1>Get Started</h1>
                     <div class="email">
                         <label for="email">E-mail</label>
                         <br>               
-                        <input type="text" id="email" name="email" placeholder="Your e-mail..">
+                        <input type="text" id="email" name="_replyto" placeholder="Your e-mail..">
                     </div>
                 </div>
                 <label for="textarea">Message</label>
-                <textarea name="textarea" id="textarea" cols="30" rows="10">Your message here ...</textarea>
+                <textarea name="message" id="textarea" cols="30" rows="10" placeholder="Your message here ..."></textarea>
```
Comment (hemantso) [inline index.html]: - [x] The `text area` should have the `max length` attribute to activate the validation on the form submit. Kindly fix it.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @leonardoeng13 and @Obote, 

Good job so far! 👍 
You still need to work on some issues to go to the next project, but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (leonardoeng13) [reply index.html]: Changed. Thanks for this review.

File: index.html (hunk 2 above)
Comment (leonardoeng13) [reply index.html:334]: Changed. Thanks for this review.

File: index.html (hunk 3 above)
Comment (leonardoeng13) [reply index.html]: Changed. Thanks for this review.

File: index.html [hunk 4]
```diff
@@ -291,23 +291,24 @@ <h1>Get Started</h1>
...
                             <label for="mobilefname">First Name</label>
                             <br>
-                            <input type="text" id="mobilefname" name="firstname" placeholder="Your name..">
+                            <input type="text" id="mobilefname" name="firstname" maxlength="30" required placeholder="Your name..">
                         </div>
                         <div class="mobile-email">
                             <label for="mobileemail">E-mail</label>
                             <br>               
-                            <input type="text" id="mobileemail" name="email" placeholder="Your e-mail..">
+                            <input type="email" id="mobileemail" name="_replyto" placeholder="Your e-mail..">
```
Comment (ahmetbozaci) [inline index.html]: - [x] Email should include required attribute. Please fix this.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️ 

## Hello @leonardoeng13 ✋🏼  Your project is great. You are almost done. However there are some issues you should fix. Please fix them then your projaect will be approved.




### ♻️ Request changes ♻️

_**Check the comment about that  under the review.**_



### Cheers and Happy coding! 💻 👏
cat on laptop



___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

File: index.html (hunk 4 above)
Comment (leonardoeng13) [reply index.html]: Please, enlighten me.

Comment (leonardoeng13) [reply index.html]: Sorry, my mistake.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔✔**
Hello @leonardoeng13 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**